src/coreclr/jit/importer.cpp

```diff
@@ -10577,7 +10581,7 @@ void Compiler::impImportBlockCode(BasicBlock* block)
 
                 op1 = gtNewLoadValueNode(layout, op1);
                 op2 = gtNewLoadValueNode(layout, op2);
-                op1 = gtNewBlkOpNode(op1, op2, ((prefixFlags & PREFIX_VOLATILE) != 0));
+                op1 = gtNewAssignNode(op1, op2);
```

Comment:
Semantic change: `volatile.` is not allowed for `cpobj` per the specification, but we did accept it here.

We could accept both `volatile.` and `unaligned.` here with `impPrefixFlagsToIndirFlags` if there is a concern about this being a breaking change.